src/stules/cards.css

```diff
@@ -0,0 +1,92 @@
+.Cards {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 0;
+}
+
+.Cards-Label {
+  padding-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.Cards-Images {
+  height: 134px;
+  width: 160px;
+  align-self: center;
+}
+
+.Cards-CodeProduct {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616878;
+  padding-top: 4px;
+}
+
+.Ratings-Stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 16px;
+  gap: 17px;
+}
+
+.Ratings-Text {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  align-self: flex-end;
+  color: #060b35;
```

Comment:
you can move this color into a variable and reuse it

Reply:
ready